<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RightForm</class>
 <widget class="QWidget" name="RightForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>339</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelB">
       <property name="text">
        <string>B:град.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Distance">
       <property name="text">
        <string>D: км</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_H">
       <property name="text">
        <string>Н:м</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Xkm">
       <property name="text">
        <string>X: км</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Ykm">
       <property name="text">
        <string>Y:км</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Номер трассы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cBNTRASS">
         <item>
          <property name="text">
           <string>новая</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Свой- чужой</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cBIFF"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>V (м/с)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_V">
         <property name="text">
          <string>1000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>H (м)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sB_H">
         <property name="maximum">
          <number>40000</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Ускорение на маневре м/с*с</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_A">
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableViewTrass">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
